Makefile

```diff
@@ -367,9 +366,9 @@ generate-manifests-docker-infrastructure: $(CONTROLLER_GEN) ## Generate manifest
 	cd $(CAPD_DIR); $(CONTROLLER_GEN) \
 		paths=./ \
 		paths=./api/... \
-		paths=./$(EXP_DIR)/api/... \
-		paths=./$(EXP_DIR)/internal/controllers/... \
-		paths=./$(EXP_DIR)/internal/webhooks/... \
+		paths=./exp/api/... \
```

Comment:
This does not exist anymore?

Reply:
(also other occurencies)

Reply:
If I see correctly now only used for CAPD

Reply:
Ah you are right, we are here in `cd $(CAPD_DIR)`